app/models/term.rb

```diff
@@ -81,7 +81,7 @@ def as_json(options = {})
 
   def max_mapping_weight
     Rails.cache.fetch("max_mapping_weight_#{configuration_profile.id}", expires_in: 10.minutes) do
```

Comment:
Maybe we don't need caching here anymore, cuz the value is persisted. Then the method could be replaced with something like this:

```ruby
delegate :max_mapping_weight, to: :configuration_profile
```